package.json

```diff
@@ -10,16 +10,16 @@
     "build": "src/*.js"
   },
   "scripts": {
-    "build": "NODE_ENV=production BABEL_ENV=production webpack --config webpack.config.js",
+    "build": "fedx-scripts webpack",
     "gc": "commit",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "lint": "eslint --ext .js --ext .jsx .",
+    "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
     "is-es5": "es-check es5 ./dist/*.js",
```

Comment:
Should we make frontend-enterprise consistent with our other edX libraries (e.g., Paragon, frontend-platform), which don't have the is-es5 check as a script or called in .travis.yml?

Reply:
yeah makes sense if the MFEs do it. George will remove travis.yml file as well